manifests/init.pp

```diff
@@ -430,7 +442,13 @@
   String $libvirt_connection = $::foreman_proxy::params::libvirt_connection,
   Boolean $bmc = $::foreman_proxy::params::bmc,
   Foreman_proxy::ListenOn $bmc_listen_on = $::foreman_proxy::params::bmc_listen_on,
-  Enum['ipmitool', 'freeipmi', 'shell'] $bmc_default_provider = $::foreman_proxy::params::bmc_default_provider,
+  Enum['ipmitool', 'freeipmi', 'shell', 'ssh'] $bmc_default_provider = $::foreman_proxy::params::bmc_default_provider,
+  String $bmc_ssh_user = $::foreman_proxy::params::bmc_ssh_user,
+  String $bmc_ssh_key = $::foreman_proxy::params::bmc_ssh_key,
```

Comment:
Should this be a `Stdlib::Absolutepath` or are relative paths allowed?

Reply:
Done.